pydatalab/pydatalab/apps/chat/blocks.py

```diff
@@ -35,14 +35,14 @@ def num_tokens_from_messages(messages: Sequence[dict]):
 
 class ChatBlock(DataBlock):
     blocktype = "chat"
-    description = "LLM Chat Block with contextual data (powered by GPT-3.5-turbo)"
+    description = "Virtual assistant"
     accepted_file_extensions: Sequence[str] = []
     __supports_collections = True
     defaults = {
-        "system_prompt": """You are a virtual assistant that helps materials chemists manage their experimental data and plan experiments. You are deployed in the group of Professor Clare Grey in the Department of Chemistry at the University of Cambridge.
+        "system_prompt": """You are a whinchat (lowercase w), virtual data managment assistant that helps materials chemists manage their experimental data and plan experiments. You are deployed in the group of Professor Clare Grey in the Department of Chemistry at the University of Cambridge.
```

Comment:
```suggestion
        "system_prompt": """You are whinchat (lowercase w), a virtual data managment assistant that helps materials chemists manage their experimental data and plan experiments. You are deployed in the group of Professor Clare Grey in the Department of Chemistry at the University of Cambridge.
```

Let's not gaslight them into thinking they can fly

Reply:
More generally we should make the prof Clare Grey bit configurable

Reply:
I suppose it would make sense to have the option of adding a config model for each app, and construct the global config from the core + all the apps. Perhaps with app-specific settings namespaced.